services/ux-backend/handlers/onboarding/clienttokens/handler.go

```diff
@@ -66,7 +63,14 @@ func handlePost(w http.ResponseWriter, r *http.Request, tokenLifetimeInHours int
 		return
 	}
 
-	if onboardingToken, err := util.GenerateClientOnboardingToken(tokenLifetimeInHours, onboardingPrivateKeyFilePath, storageQuotaInGiB, storageCluster.UID); err != nil {
+	privateKey, err := util.LoadOnboardingValidationPrivateKey(ctx, cl, namespace)
+	klog.Info("Getting the Pem key")
+	if err != nil {
+		http.Error(w, fmt.Sprintf("Failed to get private key: %v", err), http.StatusBadRequest)
+		return
+	}
+
+	if onboardingToken, err := util.GenerateClientOnboardingToken(tokenLifetimeInHours, privateKey, storageQuotaInGiB, storageCluster.UID); err != nil {
```

Comment:
This line is too long, please split over multi-lines
```suggestion
	if onboardingToken, err := util.GenerateClientOnboardingToken(
		tokenLifetimeInHours, 
		privateKey, 
		storageQuotaInGiB, 
		storageCluster.UID,
	); err != nil {
```